storage.go

```diff
@@ -286,8 +335,9 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 	// Check if this file is a link.
 	if fi.Mode()&os.ModeSymlink != 0 {
 		o.Mode |= ModeLink
-
-		target, err := filepath.EvalSymlinks(rp)
+		
+		// FIXME
```

Comment:
Please give more information for `FIXME`.

- What's the problem?
- What we did for this problem?

Reply:
Does it related to https://github.com/beyondstorage/go-service-fs/pull/64?

Reply:
> Is it related to #64?

Yes, it is.

Reply:
> > Is it related to #64?
> 
> Yes, it is.

This is not good.

Please don't assume that others know the context, and do your best to put all the information you know in a comment or at least in a PR.

Imagine if you were an external contributor and saw a line like this FIXME, how would you feel? Even if you wanted to contribute, you wouldn't be able to participate.

The most important thing for BeyondStorage: **Be open**.

Reply:
Got it, I will take care of that in the future.